packages/nextjs-cache-handler/src/handlers/cache-handlers.ts

```diff
@@ -0,0 +1,136 @@
+import { createMergedHandler } from "./cache-handler.js";
+import type {
+  CacheHandlerConfig,
+  CacheHandlersValue,
+  Handler,
+  LifespanParameters,
+} from "./cache-handler.types.js";
+import type { CacheHandler as CacheHandlerType } from "next/dist/server/lib/cache-handlers/types";
+
+export async function createCacheHandlers(
+  config: CacheHandlerConfig<CacheHandlersValue>,
+): Promise<CacheHandlerType> {
+  const handlersList: Handler<CacheHandlersValue>[] = config.handlers.filter(
+    (handler) => !!handler,
+  );
+  const memoryCache = createMergedHandler(handlersList);
```

Comment:
Why `memoryCache` as this is merged from multiple handlers, possibly redis as well? 

I think this implementation is quite naive in a sense that it only "makes it work" without addressing the actual implementation of the new interface.